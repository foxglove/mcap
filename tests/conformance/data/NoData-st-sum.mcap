version https://git-lfs.github.com/spec/v1
oid sha256:adfdbe9b9019beab03f7470fa54e3ec04793aff485cfe61bff3b1d7b56e75469
size 212
