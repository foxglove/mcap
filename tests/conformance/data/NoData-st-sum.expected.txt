Header profile="" library=""
Statistics messageCount=0 channelCount=0 attachmentCount=0 chunkCount=0 channelMessageCounts={}
SummaryOffset groupOpcode=12 groupStart=71 groupLength=0
SummaryOffset groupOpcode=10 groupStart=38 groupLength=33
SummaryOffset groupOpcode=9 groupStart=71 groupLength=0
SummaryOffset groupOpcode=7 groupStart=71 groupLength=0
Footer summaryStart=38 summaryOffsetStart=71
